performance/scripts/preScript.js

```diff
@@ -0,0 +1,46 @@
+const http = require('http');
```

Comment:
idem als clone.js? deze moet toch gewoon in util zitten of zo?

Reply:
Performance code moet niet meekomen, ik verwijder het.